st2reactor/st2reactor/cmd/sensormanager.py

```diff
@@ -36,7 +36,8 @@ def _teardown():
 
 
 def _get_all_sensors():
```

Comment:
Minor thing - maybe the method should be called `_get_all_enabled_sensors` :)

Anyway, LGTM.